### DISCUSSION TITLE
Hangouts Chat support

### DISCUSSION THREAD

**Discussion by Wouter0100** [42wim/matterbridge#1721] [New bridges suggestions]
As Google recently released Hangouts Chat to the public - with support for bots, it may be a good additional feature for matterbridge to support.

See docs here: https://developers.google.com/hangouts/chat/

**Comment by 42wim** [comment]
Looks interesting and google has a go library available for it.
But it's not a free service and the 14 day free trial seems to limited for me to test it.

If anyone can get me an account to test this, or know a way how developers can get (sandbox) access, it would be much easier for me to develop.

**Comment by Jaykul** [comment]
Well -- I don't care about Hangouts chat (I didn't even know it existed until just now), but I really appreciate matterbridge 😁 

I could give you an account or three in one of my groups @42wim if that would help 😉

**Comment by 42wim** [comment]
Interesting libs for those that wants to implement this:
* https://github.com/seibert-media/hangouts-jira-bot
* https://github.com/seibert-media/go-hangouts

**Comment by HugoPoi** [comment]
There is official API for this https://github.com/googleapis/google-api-go-client/blob/master/chat/v1/chat-gen.go
I put this one in my todo list. I will working on it when I have some time.
